orc8r/cloud/go/services/orchestrator/obsidian/models/swagger.v1.yml

```diff
@@ -1457,6 +1457,28 @@ definitions:
         minimum: 0
         maximum: 1
         example: 0.5
+      exclusion_patterns:
+        type: array
+        items:
+          type: string
+        minItems: 0
+        description: log message patterns that are excluded (regex substring match)
+    default:
+      upload_mme_log: false
+      sample_rate: 1
+      exclusion_patterns:
+        - 'GetServiceInfo'
```

Comment:
Changes look good! Last comment, I think. Is the intention here to eventually reduce these default values? In that case, I think it would be good to file an issue (and link it here in a comment)  for to migrate some of these issues to be handled directly.

Reply:
And finally, please add this configuration to the sentry doc. (can be in a different PR)

Reply:
Documentation: #11210

Reply:
I agree, the goal should also be to reduce the list and handle the corresponding errors in other ways. Added a task to track that: #11211